ambry-server/src/main/java/com.github.ambry.server/AmbryServerRequests.java

```diff
@@ -100,39 +100,6 @@
     localPartitionToReplicaMap = createLocalPartitionToReplicaMap();
   }
 
-  @Override
-  public void handleRequests(NetworkRequest request) throws InterruptedException {
```

Comment:
Why this method removed?

Reply:
AmbryServerRequests is a direct child of AmbryRequests and this method is trying to override the same method from AmbryRequests, but they are exactly the same method. So we can remove them.